pyproject.toml

```diff
@@ -18,3 +18,42 @@ typeCheckingMode = "strict"
 
 [tool.isort]
 profile = "black"
+
+[tool.ruff]
+select = ["E", "W", "I"]
+ignore = ["E741"]
+
+# Allow autofix for all enabled rules (when `--fix`) is provided.
+fixable = ["A", "B", "C", "D", "E", "F", "G", "I", "N", "Q", "S", "T", "W", "ANN", "ARG", "BLE", "COM", "DJ", "DTZ", "EM", "ERA", "EXE", "FBT", "ICN", "INP", "ISC", "NPY", "PD", "PGH", "PIE", "PL", "PT", "PTH", "PYI", "RET", "RSE", "RUF", "SIM", "SLF", "TCH", "TID", "TRY", "UP", "YTT"]
+unfixable = []
+
+# Exclude a variety of commonly ignored directories.
+exclude = [
```

Comment:
can we skip this? it's part of the default config, right? I don't see any changes according to https://beta.ruff.rs/docs/configuration/#using-pyprojecttoml

Reply:
Ok, let's remove default config.